app/mailers/notifier.rb

```diff
@@ -67,6 +67,15 @@ def statement(args)
     send_nucore_mail args[:user].email, text("views.notifier.statement.subject", facility: @facility)
   end
 
+  def offline_cancellation_notification(reservation)
```

Comment:
Glad you mentioned it; it appears it's just serializing it as yaml. I'll work that out with the extraction into its own mailer class.